js/dialogcards.js

```diff
@@ -481,6 +481,13 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
     // Check if card has been turned before
     var turned = $c.hasClass('h5p-dialogcards-turned');
 
+	//If the checkbox to use the same front image on back also is checked, show the front image only regardless of whether a back image is uploaded or not.
```

Comment:
Please change to 2 spaces instead of tabs for coding style consistency across the library